docs/command_line_reference.rst

```diff
@@ -41,68 +44,99 @@ command above).
 
 It further supports the following additional arguments:
 
-:``--checkout``: deletes the local Git repository and (re-)clones it.
-:``--compile``: (re-)compiles the build.
-:``--configure``: (re-)configures the build.
-:``--delete-source``: deletes the source directory when purging.
-:``--install``: (re-)installs the build.
-:``--purge``: deletes all related build files.
-:``--recheckout``: deletes the local build Git repository, reclones, regenerates, reconfigures, recompiles
-    and reinstalls it.
-:``--recompile``: recompiles and reinstall the build.
-:``--reconfigure``: reconfigures, recompiles and reinstalls the build.
-:``--regenerate``: (re-)regenerates the build.
-:``--reinstall``: reinstalls the build.
-:``--reregenerate``: regenerates, reconfigures, recompiles and reinstalls the build.
+:--checkout:      Deletes the local Git repository and clones it.
+:--compile:       Compiles the build.
+:--configure:     Configures the build.
+:--delete-source: Deletes the source directory when purging.
+:--install:       Installs the build files.
+:--purge:         Deletes all related build files.
+:--recheckout:    Deletes the cloned git repository, reclones, regenerates,
+                  reconfigures, recompiles, and reinstalls it.
+:--recompile:     Recompiles and reinstall the build.
+:--reconfigure:   Reconfigures the build.
+:--regenerate:    Regenerates the build.
+:--reinstall:     Reinstalls the build.
+:--reregenerate:  Regenerates, reconfigures, recompiles, and reinstalls the build.
```

Comment:
honestly I still do not understand what this command is supposed to do. If we regenerate something, then there has to be a generate step somewhere, but i do not see it?

Reply:
Read: https://simexpal.readthedocs.io/en/latest/builds.html#automated-builds
It's basically one phase before build that will be executed.

Reply:
okay, thanks :)